optimum/exporters/onnx/model_configs.py

```diff
@@ -561,6 +561,12 @@ def inputs(self) -> Dict[str, Dict[int, str]]:
         # TODO: is pixel mask needed?
         return {**super().inputs, "pixel_mask": {0: "batch_size"}}
 
+class DeformableDetrOnnxConfig(ViTOnnxConfig):
+    DEFAULT_ONNX_OPSET = 16
+
```

Comment:
he try/catch which was causing issue from what I see above was removed in 4.50 so we can add  

```suggestion
    MIN_TRANSFORMERS_VERSION = version.parse("4.50")
```